AEPAnalytics/Sources/Analytics+PublicAPI.swift

```diff
@@ -65,10 +65,7 @@ import Foundation
                 return
             }
 
-            guard let trackingIdentifier = responseEvent.data?[AnalyticsConstants.EventDataKeys.ANALYTICS_ID] as? String else {
-                completion(nil, AEPError.unexpected)
-                return
-            }
+            let trackingIdentifier = responseEvent.data?[AnalyticsConstants.EventDataKeys.ANALYTICS_ID] as? String
```

Comment:
We should handle the missing key and key present but value not a String cases. If value is not a String can still be AEPError.unexpected.
Same for visitor identifier